assets/javascripts/modules/SSO_links.js

```diff
@@ -19,25 +20,30 @@ require('jquery');
  * @returns {boolean}
  */
 module.exports = function(element, ssoUrl, ssoMethod) {
-  var $element,
-      payload,
-      clientSso,
-      serverSso,
-      destination,
-      newWindow,
-      winId;
+  var $element;
+  var payload;
+  var clientSso;
+  var serverSso;
+  var destination;
+  var winId;
+  var openInNewWindow;
+  var allowLinkCLickEvent = true;
   var elementHref;
+  var elementTarget;
   var useGet = ssoMethod === 'GET';
 
   if (element) {
     $element = $(element);
-    newWindow = !!$element.attr('target');
-    winId = element.id;
-    elementHref = element.href;
     clientSso = $element.data('sso') === true || $element.data('sso') === 'client';
     serverSso = $element.data('sso') === 'server';
 
     if (clientSso || serverSso) {
+
+      elementHref = element.href;
+      elementTarget = element.target;
+      openInNewWindow = elementTarget && elementTarget === '_blank';
```

Comment:
Can we use a value other than _blank as a test, i.e.
 `openInNewWindow = elementTarget && elementTarget != ‘_self’;`
Using a value specific to the link opens a new tab/window in the same tab/window if the tab/window is already opened